static/app/components/questionTooltip.tsx

```diff
@@ -61,4 +65,11 @@ const QuestionIconContainer = styled('span')<Pick<QuestionProps, 'size' | 'class
   }
 `;
 
+const TooltipTriggerButton = styled(Button)`
```

Comment:
Already discussed in slack, but yeah, let's not use our button and just use a regular `button` element here. So we don't have to override styles (and instead apply styles)

afaik there's nothing we would want from our button component.

We should also give it a proper `aria-label` then too.

Really all of the usages of this component should pass some kind of label prop so we can indicate to the screen reader that this button will tell you more.

Reply:
I think if we were to do this right, then the Tooltip component should ensure that whatever the trigger is, it is focusable. We could ensure this in different ways, but since tooltips are tied to a html trigger element, providing a tab index would probably be a good idea. The downside here is that this would require the children to implement the full html interface (something we pretty much never do, but should)

Rough example
```ts
function Tooltip(){
  return cloneElement(children, {tabIndex})
}

// implement full interface and pass it to the dom node
interface ButtonProps extends HTMLButtonProps {}

function Button(props){
  return <Buttton {...props}/>
}
```